modules/plot/main/coffee/charts/PieChart.coffee

```diff
@@ -79,7 +81,7 @@ class PieChart extends hx.EventEmitter
 
   render: ->
     self = this
-    selection = hx.select(@selector)
+    selection = hx.select(@selector).class('hx-pie-chart')
```

Comment:
@georgesimms This PR is waiting for the styles for `.hx-pie-chart` to be removed (the class can still be applied to the node)